### PR TITLE
bem practice pr

### PR DESCRIPTION
**Pull request by edson-pro**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+.testimonyuser__item {
+  display: flex;
+  align-items: center;
+}
+
+.testimonyuser__username {
+  margin-bottom: 8px;
+  margin-top: 0px;
+  font-size: 14px;
+}
```
Comment (hirwaaldo1) [inline main.css:254]: change this element name

**Review by hirwaaldo1 — CHANGES_REQUESTED** [review]

**Review by hirwaaldo1 — APPROVED** [review]
👌

**Review by ritha-kankindi — APPROVED** [review]
Well done keep it Up

File: index.html [hunk 2]
```diff
@@ -0,0 +1,814 @@
...
+    <div class="why">
+      <div class="why__header">
+        <h4 class="why__title">Why choose us</h4>
+        <p class="why__subtitle">
+          Problems trying to resolve the conflict between the two major realms
+          of Classical physics: Newtonian mechanics
+        </p>
+      </div>
+      <div class="why__content">
+        <div class="whycard__item">
```
Comment (Morrism1) [inline index.html]: Here we have `whycard__item` which I can read as we have a BLOCK called `whycard` and ELEMENT `item` but we don't have a BLOCK with such name.

Comment (Morrism1) [reply index.html]: Also this applies to all children elements referencing a BLOCK that we don't have.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,814 @@
...
+    <div class="testimonies">
+      <div class="testimonies__header">
+        <h4 class="testimonies__title">What Clients Say</h4>
+        <p class="testimonies__subtitle">
+          Problems trying to resolve the conflict between the two major realms
+          of Classical physics: Newtonian mechanics
+        </p>
+      </div>
+      <div class="testimonies__content">
+        <div class="testimony__item">
```
Comment (Morrism1) [inline index.html]: Here also you are referencing a BLOCK `testimony` which we don't have.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,814 @@
...
+                <rect
+                  width="110.068"
+                  height="22"
+                  fill="white"
+                  transform="translate(0.5)"
+                />
+              </clipPath>
+            </defs>
+          </svg>
+          <p class="testimony__message">
```
Comment (Morrism1) [inline index.html]: Here also you are referencing a BLOCK `testimony` which we don't have.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,814 @@
...
+              src="https://avatars.githubusercontent.com/u/78101497?v=4"
+              alt=""
+            />
+            <div>
+              <h4 class="testimonyuser__user">Regina Miles</h4>
+              <span class="testimonyuser__role">Designer</span>
+            </div>
+          </div>
+        </div>
+        <div class="testimony__item">
```
Comment (Morrism1) [inline index.html]: Here also you are referencing a BLOCK `testimony` which we don't have.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,814 @@
...
+                <rect
+                  width="110.068"
+                  height="22"
+                  fill="white"
+                  transform="translate(0.5)"
+                />
+              </clipPath>
+            </defs>
+          </svg>
+          <p class="testimony__message">
```
Comment (Morrism1) [inline index.html]: Here also you are referencing a BLOCK `testimony` which we don't have.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,814 @@
...
+            Slate helps you see how many more days you need to work to reach
+            your financial goal.
+          </p>
+          <div class="testimonyuser__item">
```
Comment (Morrism1) [inline index.html]: Here also you are referencing a BLOCK `testimonyuser` which we don't have.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,814 @@
...
+            <img
+              class="testimonyuser__img"
+              src="https://avatars.githubusercontent.com/u/78101497?v=4"
+              alt=""
+            />
+            <div>
```
Comment (Morrism1) [inline index.html]: Using elements that don't have classes is not necessary.

**Review by Morrism1 — CHANGES_REQUESTED** [review]
Resolve these issues. Most of them are concerning where you are referencing a BLOCK that not exist yet.

**Comment by rithaven** [comment]
Cordially noted!






*0787412101/0726882125*
***@***.*** ***@***.***>
Computer Engineer/ from UR-Collage of science and Technology
Full stack web developer/From wecode (Moringa school)

**Review by hirwaaldo1 — APPROVED** [review]

**Review by Morrism1 — APPROVED** [review]
Good Job @edson-pro!

**Pull request merged**